### PR TITLE
Reenable Rowan Collateral

### PR DESCRIPTION
**Pull request by AustinoBombino**
reenable rowan collateral for testing on staging

New TX form: `sifnoded tx margin update-rowan-collateral --enabled=true/false `

### CODE REVIEW

**Review by timlind — APPROVED** [review]

**Review by canercandan — APPROVED** [review]

**Review by timlind — APPROVED** [review]

**Review by canercandan — APPROVED** [review]

**Comment by AustinoBombino** [comment]
@timlind @canercandan @Jedi2002 
added a toggle parameter and tx

**Comment by canercandan** [comment]
@AustinoBombino nice thanks for working on this, I believe we should add a migration handler as well to set the default value for this new toggle correct?

**Comment by AustinoBombino** [comment]
> @AustinoBombino nice thanks for working on this, I believe we should add a migration handler as well to set the default value for this new toggle correct?

@canercandan Yup, fixed the tests and added a migrator

**Review by canercandan — APPROVED** [review]
LGTM

File: x/margin/keeper/msg_server.go [hunk 1]
```diff
@@ -344,6 +344,23 @@ func (k msgServer) UpdatePools(goCtx context.Context, msg *types.MsgUpdatePools)
 	return &types.MsgUpdatePoolsResponse{}, nil
 }
 
+func (k msgServer) UpdateRowanCollateral(goCtx context.Context, msg *types.MsgUpdateRowanCollateral) (*types.MsgUpdateRowanCollateralResponse, error) {
```
Comment (Jedi2002) [inline x/margin/keeper/msg_server.go:347]: We can probably change the update params logic , instead of adding new transactions for each param ?

**Review by Jedi2002 — APPROVED** [review]

**Pull request merged**